### ISSUE TITLE
When displaying 24 hour time, digits should be padded

### ISSUE THREAD

**Issue by mintsoft** [FossifyOrg/Clock#52]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Clock/issues) or [closed](https://github.com/FossifyOrg/Clock/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Clock/discussions) or [closed](https://github.com/FossifyOrg/Clock/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Clock/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

The default time format (9:42:01) looks odd to my British eyes without the leading zero. There doesnt seem to be anywhere to customize this or turn on 24hr clock mode etc.

### Why do you want this feature?

User preference and familiarity. As a 24hr time user, it is easier for me to recognize and read the time in consistent formats.

**Comment by Aga-C** [comment]
There should be an option to switch between 12 and 24 hour system, but there shouldn't be an option for setting leading zero. We should always pad 24 hour time, and leave 12 hour as it is.

**Comment by mintsoft** [comment]
Yeah I agree with that, so I guess the bug is that 24 hour time isn't padded currently as the clock does show (for example) `14:43:50` but not `09:02:54`
